### DISCUSSION TITLE
Welcome to BACpypes Discussions!

### DISCUSSION THREAD

**Discussion by JoelBender** [JoelBender/bacpypes#376] [General]
## 👋 Welcome!
Discussions as a place to connect with other people that are using or are interested in [BACnet](http://www.bacnet.org/) communications and implementations like BACpypes.  If you think that this forum tool is redundant when we already have [gitter](https://gitter.im/JoelBender/bacpypes), that might be true but we won't know until we use **Discussions** and see what it's like.  Apparently there is a way to convert issues to discussions, which could be good.

Ask a question, pose a puzzle, rant a bit (as long as you follow the community guidelines of being welcoming and showing respect).

**Comment by ChristianTremblay** [comment]
I like the categories. Makes a lot of sense as **issues** are not always the real intent behind a comment...
